pkg/cvo/availableupdates.go

```diff
@@ -45,14 +47,15 @@ func (optr *Operator) syncAvailableUpdates(config *configv1.ClusterVersion) erro
 		return err
 	}
 
-	updates, condition := calculateAvailableUpdatesStatus(string(config.Spec.ClusterID), proxyURL, tlsConfig, upstream, arch, channel, optr.releaseVersion)
+	current, updates, condition := calculateAvailableUpdatesStatus(string(config.Spec.ClusterID), proxyURL, tlsConfig, upstream, arch, channel, optr.releaseVersion)
 
 	if usedDefaultUpstream {
 		upstream = ""
 	}
 	optr.setAvailableUpdates(&availableUpdates{
 		Upstream:  upstream,
 		Channel:   config.Spec.Channel,
+		Current:   current,
```

Comment:
`setAvailableUpdates()` function does not use `Current` . Also I am not sure why we need `Current`?

Reply:
`Current` will allow us to set `ClusterVersion.Status.Desired.URL`.  Will get that FIXME addressed today.

Reply:
Addressed in c1de00c39d.